sdk/mongodbatlas/arm-mongodbatlas/src/api/atlasContext.ts

```diff
@@ -19,14 +20,17 @@ export interface AtlasClientOptionalParams extends ClientOptions {
   /** The API version to use for this operation. */
   /** Known values of {@link KnownVersions} that the service accepts. */
   apiVersion?: string;
+  /** Specifies the Azure cloud environment for the client. */
+  cloudSetting?: AzureSupportedClouds;
 }
 
 export function createAtlas(
   credential: TokenCredential,
   subscriptionId: string,
   options: AtlasClientOptionalParams = {},
 ): AtlasContext {
-  const endpointUrl = options.endpoint ?? options.baseUrl ?? "https://management.azure.com";
+  const endpointUrl =
```

Comment:
@v-jiaodi the version in below is wrong, please update it manually. you could refer the version in changelog.

```
const userAgentInfo = `azsdk-js-arm-mongodbatlas/1.0.0-beta.1`;
```